### DISCUSSION TITLE
Starting new plot at 3:0 ish?

### DISCUSSION THREAD

**Discussion by Yamanipanuchi** [ericaltendorf/plotman#941] [Q&A]
I read through the instructions best I could but unfortunately I still dont see if this is possible?

I have a plot going, It reaches phase 3:0, I would like a new plot to start. Is that possible?

**Comment by altendky** [comment]
That would be done via the phase limit.  The most common is limiting to some number of plots in phase 1 by specifying a limit of 2:0.  In your case you are asking for 3:0.  See below.

But, maybe say a bit more about your scenario so we can make sure we get it right.  Keep in mind that the _tmpdir_ stagger limits are per tmp dir, not global.  Also, what plotter are you using?  On the two plotters I mostly used I ended up with madmax and a phase 1 (2:0) stagger since it was only able to fill the CPU in phase 1.

https://github.com/ericaltendorf/plotman/blob/1b7dfba139ed03d3c32a2f04f6011e03bfb1f442/src/plotman/resources/plotman.yaml#L97-L110

**Reply by Yamanipanuchi** [reply]
IM using MadMax, I have two 500gb SSD's I am plotting on. By the two config setting your have above that would indicate that when plot on /tmp1 reaches 2:1 it will start another plot. Is that another plot on that same drive or on another drive?

```taml
        # Five is the final move stage of madmax
        tmpdir_stagger_phase_major: 5
        tmpdir_stagger_phase_minor: 0
        # Optional: default is 1
        tmpdir_stagger_phase_limit: 1

        # Don't run more than this many jobs at a time on a single temp dir.
        # Increase for staggered plotting by chia, leave at 2 for madmax sequential plotting
        tmpdir_max_jobs: 2
```

That is what I have, But that was what was there by default. I am going to guess because there's a 5 in the major it wont start another plot unit the current plot is completed in it entirety?

**Reply by altendky** [reply]
Satisfying one limit won't "start another plot" necessarily.  All configurations are limitations and if they are all satisfied then a plot will start.  There's no configuration that forces a plot to start.  Anyways...  are they matched 500gb SSDs?  If so I might raid them and then go with the stagger I mentioned.  You end up with three or four plot jobs simultaneously.

In case a visualization helps, this is the current plot progress on my plotter.  There's some stuff about it that makes the runtimes inconsistent plot to plot, but on another system I used to run it was more consistent.

![image](https://user-images.githubusercontent.com/543719/143134559-d79e2a58-f2aa-45c1-8519-f341d4de5f16.png)

But, if you stick with a pair of separate SSDs, instead of merging them in a RAID 0, then yes, the phase limits are per drive.  If you took the config you shared and reduced the phase to 3:0 then you would have one plot going on each tmp SSD with a phase under 3:0 and another (probably just one) in a phase of 3:0 or higher.

**Reply by Yamanipanuchi** [reply]
Perfect, That actually answers the question perfectly. Ill need to consider RAID 0. Thank you!
